src/OpenTK/Platform/Windows/XInputJoystick.cs

```diff
@@ -434,7 +435,7 @@ private Delegate Load(ushort ordinal, Type type)
 
             private Delegate Load(string name, Type type)
```

Comment:
This is probably worth making a generic method.

Reply:
prob. out of scope